consensus/src/util/time_service.rs

```diff
@@ -10,6 +10,8 @@ use std::{
 use thiserror::Error;
 use tokio::{runtime::Handle, time::delay_for};
 
+// REVIEW: note the existence of a TimeService in secure/time/src/lib.rs
```

Comment:
they copy us!

Reply:
The hope would be to unify them at some point in time, this one has support for async / tokio :(